src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const obj = {};
+  const arr = sourceString
```

Comment:
Avoid using variable names like 'arr' as it decreases code readability. Consider using a more descriptive name for the array.